src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,10 +1,26 @@
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd:(movie:Movie)=> void
+};
+
+export const NewMovie:React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
   const [count] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
+  const [disabledButton, setdisabledButton] = useState(true);
```

Comment:
camelCase:upside_down_face:
```suggestion
  const [disabledButton, setDisabledButton] = useState(true);
```

Reply:
Please, fix @dffUqp comment